apis/python/src/tiledbsoma/__init__.py

```diff
@@ -115,6 +115,7 @@
     get_storage_engine,
     show_package_versions,
 )
+from ._index_util import build_index
```

Comment:
Are the functions in `_util.py` (which includes `_index_util.py`) supposed to be for internal only? If so, we might want to move these into a `_indexer.py`.

Reply:
As you mentioned, lets address these in a separate PR if that's ok